src/components/profile.component.js

```diff
@@ -72,6 +72,7 @@ class Profile extends React.Component {
     var postBody = {
       firstName: this.state.firstName,
       lastName: this.state.lastName,
+      password: "sss",
```

Comment:
Is this updating the user's password to `"sss"`?

Reply:
I will update it. That's for testing purposes. Password property is no longer required. Thanks for the catch. 👍

Reply:
Ah, okay, just making sure. Everything else looks really good! Thanks, Harish!